### PR TITLE
impr(UTAPI-55): Improve warp 10 request error logging

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: libV2/server/API/metrics/listMetrics.js [hunk 1]
```diff
@@ -28,47 +28,55 @@ async function listMetric(ctx, params) {
...
+                        params: {
+                            start: convertTimestamp(start).toString(),
+                            end: convertTimestamp(end).toString(),
+                            labels,
+                            node: warp10.nodeId,
+                        },
+                        macro: 'utapi/getMetrics',
+                    };
+                    return warp10.exec(options);
+                }, error => ctx.logger.error('error during warp 10 request', { error }));
```
Comment (jonathan-gramain) [inline libV2/server/API/metrics/listMetrics.js]: I'd suggest adding the request params in the log

**Review by jonathan-gramain — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

**Pull request merged**